src/components/Post/Comment/CommentsContainer/CreationLabelForComments.tsx

```
@@ -0,0 +1,216 @@
+			key: 'sentiment',
+			label: getSentimentLabel(sentiment as ESentiment) || null
+		}
+	];
+
+	return (
+		<div className={`${className} flex w-full flex-wrap items-center justify-between`}>
+			<div className={`flex text-xs ${isRow ? 'flex-row' : 'flex-col'} gap-1 max-sm:gap-1 md:flex-row md:items-center`}>
+				<NameLabel
+					defaultAddress={defaultAddress}
+					username={username}
+					truncateUsername={isMobile ? true : false}
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)